pkg/apis/batch/types.go

```diff
@@ -138,6 +138,18 @@ type JobSpec struct {
 
 	// Describes the pod that will be created when executing a job.
 	Template api.PodTemplateSpec
+
+	// ttlSecondsAfterFinished limits the lifetime of a Job that has finished
+	// execution (either Complete or Failed). If this field is set, once the Job
+	// finishes, it will be deleted after ttlSecondsAfterFinished expires. When
```

Comment:
might be worth nuancing "it will be deleted after ttlSecondsAfterFinished expires" into "`ttlSecondsAfterFinished` after the Job finishes, it is eligible to be automatically deleted" since there's no guarantee the deletion happens exactly at that moment.

Reply:
Updated